content/applications/hr/fleet/service.rst

```diff
@@ -86,61 +298,219 @@ clicked, sort the data in different ways.
+repair price, from lowest to highest.
 
-The default sorting is descending alphabetical order (A to Z). Click the arrow again to reverse the
-order, and go in reverse alphabetical order (Z to A). The one exception to this sorting is the
-default :guilabel:`Date` column, which sorts the information in chronological order (January to
-December) instead of alphabetical order.
+Add a service
+*************
+
+To add a service record from the list view, click the :guilabel:`New` button, and a service form
+loads. :ref:`Enter all the information <service/service-form>` on the service form.
+
+The form automatically saves as data is entered.
 
 Kanban view
 ~~~~~~~~~~~
 
 To view services by their stage, click on the :guilabel:`Kanban icon`, which is the second icon in
-the top right, and appears as four black squares in a cube.
+the top-right corner, and appears as two different length bars beneath a line.
+
+All services with the same status appear in the corresponding Kanban column, from
+:guilabel:`Running` to :guilabel:`Canceled`.
+
+The number of repairs for each type of service appears in the far-right of each Kanban column
+header.
+
+The status of any scheduled activities appears in the color-coded bar beneath each Kanban column
+title. Repairs with activities scheduled in the future appear green, activities due today appear
+yellow, overdue activities appear red, and repairs with no activities scheduled appear gray.
+
+Each Kanban card displays a color-coded :guilabel:`(activity icon)`, indicating what type of
+activity is scheduled, and the status. For example, a scheduled phone call displays a phone icon.
+The status colors correspond to the colors in the status bar. Click on an :guilabel:`(activity
+icon)` to view the details of the activity.
+
+The length of the color bar is proportionate to the number of corresponding activities with that
+status. Hover over a color section to reveal the number of service records in that specific
+grouping.
+
+.. image:: service/kanban.png
+   :align: center
+   :alt: The service records presented in a Kanban view.
+
+To view **only** the records with a specific status, click the desired :guilabel:`color bar`
+section. The background color for the column changes to a pale hue of the same color (either green,
+yellow, red, or gray), and the color bar appears striped instead of solid. **Only** repairs and
+services with the selected status appear in the column.
+
+.. image:: service/kanban-status.png
+   :align: center
+   :alt: The oil change service records showing only repairs with past-due activities.
+
+.. _service/schedule-activity:
+
+Schedule activities
+*******************
+
+To schedule an activity for a repair or service from the Kanban view, click the :guilabel:`activity
+icon` in the lower-right corner of the service record, and click :guilabel:`+ Schedule an activity`.
+A :guilabel:`Schedule Activity` pop-up window appears.
```

Comment:
The only reason I didn't want to do the clock was, you can click on any of the icons that appear-  phone, to-do, etc. So since it is an icon that changes, I didn't want to call out something that might not appear. So I removed the gui entirely, and put in a note that it can look like many icons. I think that works- do you?